js/lib/ilib.js

```diff
@@ -137,8 +137,7 @@ ilib._getPlatform = function () {
                 return ilib._platform;
             }
         } catch (e) {}
-
-        if (typeof(global) !== 'undefined' && global.process && global.process.versions && global.process.versions.node && typeof(module) !== 'undefined') {
+	if (typeof(global) !== 'undefined' && global.process && ((global.process.versions && global.process.versions.node && typeof(module) !== 'undefined') || global.process.argv[0].includes("iotjs"))) {
```

Comment:
I think maybe using `typeof(process.iotjs) !== 'undefined'` might be a bit better because that way, it doesn't depend on the executable name including "iotjs" in it which could easily be renamed.

Also, the same change should be made to the https://github.com/ilib-js/ilib-env project.

I tried running the unit tests on iotjs and they failed because iotjs didn't support the "vm" module.

Reply:
Also, since nodejs and iotjs are not really the same platform, shouldn't we return a different platform string for it? That way, you can do different things based on the results. (Like maybe an IotJsLoader which is separate from the NodeLoader)